test/test_integration.py

```diff
@@ -456,6 +456,20 @@ def render_content(tab):
         time.sleep(2)
         self.assertEqual(tabs_content.text, 'Test content 1')
 
+    
+    def test_tabs_with_children_undefined(self):
```

Comment:
🐅 I think this test could be improved, right now it doesn't do much. Maybe add the tabs in a callback triggered by a button ?

Reply:
This test tests if a Tabs component still renders even if `children` is undefined. Previously, if `children` was not set, it would break, thus throwing an error, and thus failing this test. This PR fixes #265 as mentioned above.